### PR TITLE
Membership Purchase Flow UI

### PR DESCRIPTION
**Pull request by FrederikBolding**
- Updated lifetime membership to use ETH contract
- Added general membership styling to confirmation and receipt panel
- Updated membership purchase form to include correct fields etc.

### CODE REVIEW

File: common/v2/features/PurchaseMembership/components/MembershipPurchaseForm.tsx [hunk 1]
```diff
@@ -17,15 +17,17 @@ import { fetchGasPriceEstimates } from 'v2/services/ApiService';
 import { getNonce } from 'v2/services/EthService';
 import { EtherUUID } from 'v2/utils';
 
-import { MembershipPurchaseState, ISimpleTxFormFull } from '../types';
+import { MembershipPurchaseState, MembershipSimpleTxFormFull } from '../types';
+import { IMembershipId, IMembershipConfig, MEMBERSHIP_CONFIG } from '../config';
+import MembershipDropdown from './MembershipDropdown';
+import { getAccountsWithAssetBalance } from 'v2/features/SwapAssets/helpers';
```
Comment (blurpesec) [inline common/v2/features/PurchaseMembership/components/MembershipPurchaseForm.tsx]: reorder l'imports

File: common/v2/features/PurchaseMembership/components/MembershipReceiptBanner.tsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import { translateRaw } from 'v2/translations';
+import { COLORS, SPACING, FONT_SIZE } from 'v2/theme';
+import { IMembershipConfig } from '../config';
+import { Typography } from 'v2/components';
```
Comment (blurpesec) [inline common/v2/features/PurchaseMembership/components/MembershipReceiptBanner.tsx]: reorder l'imports

File: common/v2/features/PurchaseMembership/components/MembershipReceiptBanner.tsx [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+const ExpiryWrapper = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+`;
+
+const MembershipReceiptBanner = ({ membershipSelected }: Props) => {
+  const today = new Date();
+  const expiration = new Date(today.getTime() + 86400000 * membershipSelected.durationInDays);
```
Comment (blurpesec) [inline common/v2/features/PurchaseMembership/components/MembershipReceiptBanner.tsx]: Should extract this out to a config file methinks so we can use in >1 place

File: common/v2/features/PurchaseMembership/components/MembershipSelectedBanner.tsx [hunk 4]
```diff
@@ -0,0 +1,83 @@
+import React, { useContext } from 'react';
+import styled from 'styled-components';
+
+import { translateRaw } from 'v2/translations';
+import { COLORS, SPACING, FONT_SIZE } from 'v2/theme';
+import { IMembershipConfig } from '../config';
```
Comment (blurpesec) [inline common/v2/features/PurchaseMembership/components/MembershipSelectedBanner.tsx]: reorder l'imports

File: common/v2/features/PurchaseMembership/config.ts [hunk 5]
```diff
@@ -5,6 +5,7 @@ import threemonthsIcon from 'assets/images/membership/membership-threemonths.svg
 import sixMonthsIcon from 'assets/images/membership/membership-sixmonths.svg';
 import twelveMonthsIcon from 'assets/images/membership/membership-twelvemonths.svg';
 import lifetimeIcon from 'assets/images/membership/membership-lifetime.svg';
+import { DAIUUID, EtherUUID } from 'v2/utils';
```
Comment (blurpesec) [inline common/v2/features/PurchaseMembership/config.ts]: reorder l'imports

**Comment by blurpesec** [comment]
I'm also getting an error on `/membership/buy`: 
image

**Comment by FrederikBolding** [comment]
I think the error in the console is due to you not having run `yarn install` to update to the latest version of Formik. If not please let me know!

Have fixed other comments too.

**Pull request merged**